SelfReflection-1-MWoodhouse.qmd

```diff
@@ -0,0 +1,58 @@
+
+**This Course**
+
+5.  Please tell me a little about your prior experience with R ("I use it all the time", "I am still learning", "I've never used it before"). What do you like most about working with R? What are the biggest obstacles for you when working with R?
+
+    - In my previous position with the Phoenix NPR affiliate, I used R regularly for mapping and other visualiation work. However, it has been a challenge to bake it into my regular at BSPR, which is something I am hoping to change. 
+
+6.  Describe your experience working with spatial data and GIS (either in R, or ArcGIS, or QGIS etc.). How do you tend to use spatial data (e.g., making maps, desiging field sampling, buidling stastical models)?
+
+    - Mostly mapping. Leaflet is probably package I use most consistently. 
+
+7.  How do you feel about taking this course (e.g., neutral, positive, negative) and why (e.g., aspects you are worried about, or looking forward to)?
+
+    - I'm very excited. My worries are mostly to do with the responsibilities I'll be juggling. 
+
+8.  After reviewing the syllabus, give some thought to your objectives for this course. Which parts of this course are going to be most useful to your research? What skills do you really want to focus on during the semester? Once you've given this some thought, craft 3-5 SMART objectives for you in this course. SMART objectives are **S**pecific (e.g., I want to make maps in R), **M**easurable (e.g., I want to build *3 maps displaying different elements of a location* in R), **A**chievable (e.g., I want to build 3 maps displaying different elements of a location *using ggplot*), **R**elevant (e.g., I want to build 3 maps displaying *social and ecological* elements of a location using ggplot), and **T**ime-bound (e.g., I want to build 3 maps displaying *social and ecological* elements of a location using ggplot *for the final project*). Try to think about your overall objectives for the course so that we can use to assess your final project. Remember, these objectives should be based in where you are now and where you'd like to be when the course ends. **If you're not sure how to answer this, schedule a meeting with me to chat about it**.
+
+    - I'm hoping to use this course to develop a number of scripts capable of reading, analyzing and visualizing state and regional results for the upcoming November election. Ideally, this would be within a standalone website that could be used by my station during the election. Obviously the election comes before the end of the semester. For the final project, I think it would be cool to do some additional statistical analysis, for example examining the relationship between things like household income and voting results. 
```

Comment:
I'm not sure what experience you have with publishing websites, but you can build websites with Quarto and R Shiny, and you should be able to publish one website with your GitHub account if your station doesn't have another way to publish websites. Let me know if you'd like some support with that, and I can try to hunt down some resources.